src/components/Components/LinkedSecretsListView/LinkedSecretsListHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import { createTableHeaders } from '../../../shared/components/table/utils';
+
+export const linkedSecretsTableColumnClasses = {
```

Comment:
The width of the columns are oversized?  whether we need to the sum of the width <=100?

Reply:
updated!